src/fp6.rs

```diff
@@ -110,6 +110,13 @@ impl Fp6 {
         }
     }
 
+    #[inline]
+    // #[cfg(target_os = "zkvm")]
```

Comment:
Question: Can we uncomment this and make this only in the `zkvm` like the other `_inp` functions?